packages/components/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,205 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconDotsHorizontal } from '@frontify/fondue-icons';
+import * as RadixTabs from '@radix-ui/react-tabs';
+import {
+    createContext,
+    forwardRef,
+    useCallback,
+    useContext,
+    useEffect,
+    useRef,
+    type ForwardedRef,
+    type ReactNode,
+} from 'react';
+
+import { useControllableState } from '#/hooks/useControllableState';
+
+import { Button } from '../Button/Button';
+import { Dropdown } from '../Dropdown/Dropdown';
+
+import { useTabTriggers } from './hooks/useTabTriggers';
+import styles from './styles/tabs.module.scss';
+import { type TabTrigger } from './types';
+
+export type TabsRootProps = {
```

Comment:
how do you feel about having the types next to the components they belong to rather than separating with all types first and then all components below? I think it makes for an easier read, wdyt?

Reply:
imo because the types are all very short, it is very easy to read this way.

I'm also okay with having them at the top if you feel that would bring us a improvement

Reply:
yes in this case it's not a big deal but it's one of those things where consistency helps a bit. I think this is a good topic of discussion for our focus week, that and the use of implicit returns which sometimes I feel we do and other times not and I'm not 100% clear on what our philosophy is on that